portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,12 +46,15 @@
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    int maxComments = Integer.parseInt(UtilityFunctions.getFieldFromResponse(request, "maxcomments", defaultMaxComment));
+    int maxComments = Integer.parseInt(UtilityFunctions.getFieldFromResponse(request, "maxcomments"
```

Comment:
Sorry you couldn't auto format your Java code :( 

According to https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break, the comma stays attached to the preceding element. 

In your case I think it would look nicer if you line break right after `getFieldFromResponse`, so that the arguments stays together. Something like:
```
int maxComments = Integer.parseInt(UtilityFunctions.getFieldFromResponse(
    request, "maxcomments", defaultMaxComment));
```
You can also play around with other ways for line breaking and see which one you like. 
```
int maxComments = Integer.parseInt(
    UtilityFunctions.getFieldFromResponse(request, "maxcomments", defaultMaxComment));
```
```
int maxComments = Integer.parseInt(
    UtilityFunctions.getFieldFromResponse(
        request, "maxcomments", defaultMaxComment));
```
```
int maxComments = Integer.parseInt(
    UtilityFunctions.getFieldFromResponse(
        request, 
        "maxcomments", 
        defaultMaxComment));
```